### ISSUE TITLE
Toggle All has no Spacing

### ISSUE THREAD

**Issue by Chibicon** [zixaphir/appchan-x#865]
I haven't changed any settings so I dunno.

http://i.imgur.com/8vfrADo.png

**Comment by zixaphir** [comment]
I removed the spacing. Do you want it back?

**Comment by DaYuffie** [comment]
I think mine is a similar issue caused by removing spacing. Custom board titles don't work either.
It would be nice if you brought it back. Why was it removed in the first place?

**Comment by Chibicon** [comment]
Yes I would love it back. Some people may prefer no spacing though so if you could make it an option that would be even better.

**Comment by DaYuffie** [comment]
Still doesn't seem to be working correctly after updating.
http://puu.sh/eSzmD.png is what I've entered in the advanced tab.
http://puu.sh/eSzu9.png This is what shows up though.

**Comment by zixaphir** [comment]
That's a different issue. I also haven't pushed out a new version yet.

**Comment by Chibicon** [comment]
Did this ever get implemented?
